impeller/renderer/render_pass.cc

```diff
@@ -3,14 +3,16 @@
 // found in the LICENSE file.
 
 #include "impeller/renderer/render_pass.h"
+#include "impeller/tessellator/tessellator.h"
 
 namespace impeller {
 
 RenderPass::RenderPass(std::weak_ptr<const Context> context,
                        const RenderTarget& target)
     : context_(std::move(context)),
       render_target_(target),
-      transients_buffer_(HostBuffer::Create()) {}
+      transients_buffer_(HostBuffer::Create()),
+      tessellator_(Tessellator::CreateTessellatorContext()) {}
```

Comment:
I think this is still getting recreated once per frame? Would like to reuse it across frames

Reply:
Render pass is at least once per frame.

Reply:
Moved to ContentsContext